crypto/ec/ec_mult.c

```diff
@@ -211,6 +211,10 @@ static int ec_mul_consttime(const EC_GROUP *group, EC_POINT *r,
         || (bn_wexpand(r->Z, group_top) == NULL))
         goto err;
 
+    /* applying the EC point blinding if supported*/
+    if (!ec_point_blind_coordinates(group, s, ctx))
+        goto err;
```

Comment:
Commentary is confusing, because in combination with code it kind of suggests that if blinding is not supported, then it's an error condition.